.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  lint:
```

Comment:
(Optional)
The new workflow doesn't run shellcheck like the Circle CI one did, however I guess we are moving away from shell scripts on the most part, so maybe this is intentional?

If you wanted to still run shellcheck (in this same lint job; don't think it would warrant its own job), it's installed in the base GitHub Actions image, so wouldn't need to be installed specially.

Reply:
Yeah, I left it out intentionally :) I'll put some time into figuring out how to do testing for meta-buildpacks with `libcnb-test` and then nuke all bash and ruby from this repo.